backend/src/main/java/quarano/actions/DiaryEventListener.java

```diff
@@ -104,28 +101,75 @@ private boolean determineWhetherEntryMostRecent(TrackedPersonIdentifier personId
 
 	/**
 	 * manage the list of characteristic symptoms - the resulting action depends
+	 * 
 	 * @param entry
 	 */
 	void handleDiaryEntryForCharacteristicSymptoms(DiaryEntry entry) {
 
 		var person = entry.getTrackedPersonId();
+		var type = cases.findTypeByTrackedPerson(person).orElse(CaseType.CONTACT);
+
+		if (type == CaseType.INDEX) {
+			return;
+		}
+
 		var actionItems = DiaryEntryActionItems
 				.of(items.findUnresolvedByDescriptionCode(person, DescriptionCode.FIRST_CHARACTERISTIC_SYMPTOM), items::save);
 
-		// Characteristic symptom reported
 		if (entry.getSymptoms().hasCharacteristicSymptom()) {
-			actionItems.adjustFirstCharacteristicSymptomTo(entry, it -> createItem(person, it));
+			actionItems.adjustFirstCharacteristicSymptomTo(entry, it -> createCharacteristicItem(person, it));
 		} else {
 			actionItems.removeFirstCharacteristicSymptomFrom(entry, () -> diaryManagement.findDiaryFor(person),
-					it -> createItem(person, it));
+					it -> createCharacteristicItem(person, it));
 		}
 	}
 
-	private DiaryEntryActionItem createItem(TrackedPersonIdentifier person, DiaryEntry entry) {
+	private DiaryEntryActionItem createCharacteristicItem(TrackedPersonIdentifier person, DiaryEntry entry) {
 		return new DiaryEntryActionItem(person, entry, ItemType.MEDICAL_INCIDENT,
 				Description.of(DescriptionCode.FIRST_CHARACTERISTIC_SYMPTOM));
 	}
 
+	/**
+	 * manage the list of conspicuous symptoms - the resulting action depends
+	 * 
+	 * @param entry
+	 */
+	void handleDiaryEntryForConspicuousSymptoms(DiaryEntry entry) {
+
+		var person = entry.getTrackedPersonId();
+		var type = cases.findTypeByTrackedPerson(person).orElse(CaseType.CONTACT);
+
+		var actionItems = DiaryEntryActionItems
+				.of(items.findUnresolvedByDescriptionCode(person, DescriptionCode.CONSPICUOUS_SYMPTOM), items::save);
+
+		Symptoms symptoms = entry.getSymptoms();
+		if (type == CaseType.INDEX
+				? symptoms.hasConspicuousSymptomeAtIndex()
+				: symptoms.hasConspicuousSymptomeAtContact()) {
+
+			actionItems.adjustConspicuousSymptomTo(entry, type, it -> createConspicuousItem(person, type, it));
+		} else {
+			actionItems.removeConspicuousSymptomFrom(entry, type, () -> diaryManagement.findDiaryFor(person),
+					it -> createConspicuousItem(person, type, it));
+		}
+	}
+
+	private DiaryEntryActionItem createConspicuousItem(TrackedPersonIdentifier person, CaseType type, DiaryEntry entry) {
```

Comment:
Can we move the private methods to the bottom of the class as described [here](https://quarano.de/internal-dev/api/docs/index.html#_organizing_methods)?

Reply:
Sorry, done